addons/account/views/account_invoice_view.xml

```diff
@@ -47,18 +47,30 @@
                     <field name="uom_id" groups="uom.group_uom"/>
                     <field name="price_unit"/>
                     <field name="discount" groups="base.group_no_one"/>
-                    <field name="price_subtotal"/>
+                    <field name="price_subtotal" groups="account.group_show_price_subtotal"/>
+                    <field name="price_total" groups="account.group_show_price_total"/>
                     <field name="currency_id" invisible="1"/>
                 </tree>
             </field>
         </record>
 
+        <!--
+            NOTE: This view uses "parent" and will not work standalone.
```

Comment:
What does it mean ?

Reply:
The view will only work if it's called as the default view to display a one2many (showing invoice lines from inside the invoice form, where the invoice is the "parent").

If there is an action that is created using this view directly (eg. showing all the invoice lines independently of their invoices) it will not work, because there is the use of `parent.` inside the view, which will not exist in that case.